packages/components/src/images/icons/tor.svg

```diff
@@ -1 +1,3 @@
-<svg xmlns="http://www.w3.org/2000/svg" fill-rule="evenodd" stroke-linejoin="round" stroke-miterlimit="2" clip-rule="evenodd" viewBox="0 0 19 19"><path d="M9.516 16.045v-.97c3.022-.052 5.48-2.553 5.48-5.575 0-3.022-2.458-5.523-5.48-5.576v-.969c3.585.009 6.53 2.96 6.529 6.545 0 3.584-2.945 6.536-6.529 6.545zm0-3.395a3.166 3.166 0 003.151-3.15 3.166 3.166 0 00-3.151-3.151v-.97A4.14 4.14 0 0113.554 9.5a4.14 4.14 0 01-4.038 4.12v-.97zm0-4.846c.93 0 1.696.765 1.696 1.696 0 .93-.765 1.696-1.696 1.696V7.804zM1.5 9.5c0 4.389 3.61 8 8 8 4.389 0 8-3.611 8-8 0-4.39-3.611-8-8-8-4.39 0-8 3.61-8 8z"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none" viewBox="0 0 24 24">
+    <path fill="gray" fill-rule="evenodd" d="M11 3.055C6.5 3.552 3 7.367 3 12s3.5 8.448 8 8.945V3.055zm2 17.89c4.5-.497 8-4.312 8-8.945s-3.5-8.448-8-8.945v1.343c3.762.49 6.667 3.707 6.667 7.602 0 3.896-2.905 7.113-6.667 7.602v1.343zM12 1C5.925 1 1 5.925 1 12s4.925 11 11 11 11-4.925 11-11S18.075 1 12 1zm1 6.782v-1.36c2.653.472 4.667 2.79 4.667 5.578 0 2.789-2.014 5.107-4.667 5.58v-1.363c1.911-.452 3.333-2.168 3.333-4.217 0-2.05-1.422-3.766-3.333-4.218zM14.333 12c0 .93-.545 1.734-1.333 2.108V9.891c.788.374 1.333 1.178 1.333 2.109z" clip-rule="evenodd"/>
+</svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
    <path fill-rule="evenodd" d="M11 3.055C6.5 3.552 3 7.367 3 12s3.5 8.448 8 8.945V3.055zm2 17.89c4.5-.497 8-4.312 8-8.945s-3.5-8.448-8-8.945v1.343c3.762.49 6.667 3.707 6.667 7.602 0 3.896-2.905 7.113-6.667 7.602v1.343zM12 1C5.925 1 1 5.925 1 12s4.925 11 11 11 11-4.925 11-11S18.075 1 12 1zm1 6.782v-1.36c2.653.472 4.667 2.79 4.667 5.578 0 2.789-2.014 5.107-4.667 5.58v-1.363c1.911-.452 3.333-2.168 3.333-4.217 0-2.05-1.422-3.766-3.333-4.218zM14.333 12c0 .93-.545 1.734-1.333 2.108V9.891c.788.374 1.333 1.178 1.333 2.109z" clip-rule="evenodd"/>
</svg>
```
I prefer SVG without default colour. It can cause potential colour bugs.

Reply:
4dc13ab